build/readme.loader.cts

```diff
@@ -0,0 +1,8 @@
+// Can't use modules because this is a bizarropackscript world
+module.exports = async function ReadmeLoader () {
+	const {getFeatures, getFeaturesMeta} = await import("./readme-parser.js");
+	return `
+		export const features = ${JSON.stringify(getFeatures())};
```

Comment:
This in reality reads the list of enabled features from the `refined-github.ts`. It _could_ be replaced by a hook that reads the files directly from webpack’s tree… but this function is also used by the feature checker…